projects.go

```diff
@@ -12,12 +12,17 @@ type ProjectService interface {
 	Update(*ProjectUpdateRequest) (*Project, *Response, error)
 	Delete(string) (*Response, error)
 	ListIPAddresses(string) ([]IPAddress, *Response, error)
+	ListStorages(string) ([]Storage, *Response, error)
```

Comment:
I think we should call this `ListStorage` or probably even better would be `ListVolumes` and have the type be called `Volume`, right?